app/templates/direct-award/end-search.html

```diff
@@ -29,49 +29,61 @@
 {% endblock %}
 
 {% block main_content %}
+{% if errors %}
+  <div class="validation-masthead" aria-labelledby="validation-masthead-heading" aria-role="group" tabindex="-1">
+    <p>
+      You need to <a href="#user_understands">confirm that you’ve finished editing your search</a> before you
+      export your results.
+    </p>
+  </div>
+{% endif %}
+
 
 <div class="end-search-page">
   <div class="grid-row">
-    <div class="column-two-thirds">
+    <div class="column-two-thirds dmspeak">
```

Comment:
What does `dmspeak` do here?

Reply:
Ahhhh dmspeak. Long story behind it I think but I think all you need to know is it's a dumping ground of all kinds of more "conversational" styles we use...